WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationWizardContent.swift

```diff
@@ -245,15 +272,19 @@ extension SiteInformationWizardContent: UITableViewDataSource {
     }
 
     private func configure(_ cell: InlineEditableNameValueCell, index: IndexPath) {
-        if Rows.title.matches(index.row) {
+        if Row.title.matches(index.row) {
             cell.nameLabel.text = TableStrings.site
             cell.valueTextField.attributedPlaceholder = attributedPlaceholder(text: TableStrings.site)
+            cell.valueTextField.delegate = self
+            cell.valueTextField.returnKeyType = .next
```

Comment:
I'm going to leave a comment about this in the separate ticket for the design issue